tools/run_tests/run_interop_tests.py

```diff
@@ -1401,7 +1425,7 @@ def aggregate_http2_results(stdout):
             for language in languages:
                 for test_case in _TEST_CASES:
                     if not test_case in language.unimplemented_test_cases():
-                        if not test_case in _SKIP_ADVANCED + _SKIP_COMPRESSION + _SKIP_SPECIAL_STATUS_MESSAGE:
+                        if not test_case in _SKIP_ADVANCED + _SKIP_COMPRESSION + _SKIP_SPECIAL_STATUS_MESSAGE + _ORCA_TEST_CASES:
```

Comment:
This looks needed, because this is cloud_to_prod, and java java_okhttp clients will be running this.

ref: test failed: https://source.cloud.google.com/results/invocations/d88be85d-7786-45fa-9730-73e7bd23fbdf/log